test/selenium/lib/view/log_in/accessors.js

```diff
@@ -0,0 +1,24 @@
+var By = require('selenium-webdriver').By;
+var Accessors = require('../accessors');
+
+
+function LoginAccessors() {
+  Accessors.apply(this, arguments);
+}
+
+LoginAccessors.prototype = Object.assign({
+
+  get passwordField() {
+    return this.waitForElement(By.id('signup-pwd1'));
+  },
```

Comment:
I'm not too sure to understand why. Would it be easier if we used the SignUpView instead?